.travis.yml

```diff
@@ -1,24 +1,19 @@
 language: python
+python:
+  - "3.5"
 install:
   - pip install coveralls tox
 script:
   - tox
 env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
   - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
   - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
   - TOXENV=py33-django17
-  - TOXENV=py33-django18
   - TOXENV=py34-django17
+  - TOXENV=py33-django18
```

Comment:
I've updated `.travis.yml` with both the sorting, and the generation code.

> P.S.: I don't think it's easy to "automate" this in a sensible way

I've posted a link to a scenario where this has been achieved already, with just a couple of lines.

> I don't like crypographic shell scripting though.

I linked to a very clear shell script that deals with this. A simple script is infinitely superior to having to remind devs to manually update files after committing certain files.

We can, of course, rewrite it into python, if you like, though it seems a bit of an overkill for just a one-line `sed`.